web/app/scripts/src/directives/proportion-field.es.js

```diff
@@ -53,14 +66,19 @@ module.directive('proportionField', [
                         return;
                     }
 
-                    const newSum = ProportionField.calculateTotal(newModel)
-                    scope.total = newSum
+                    const newSum = ProportionField.calculateTotal(newModel);
+                    scope.total = newSum;
 
                     if (newSum > 100) {
                         scope.total = 100
                         const changedIndex = ProportionField.getChangedFieldIndex(newModel, oldModel);
                         scope.model[changedIndex] -= newSum - 100;
                     }
+
+                    scope.fields.forEach((val, i) => {
+                        scope.optionsPerSlider[i].maxLimit = 100 - newSum + scope.model[i];
```

Comment:
Inside the previous `if` the model is being changed on the altered field and that will trigger a `watch`, so it will be rerun with a correct sum.